src/test/java/clear_all_use_case/ClearAllInteractorTest.java

```diff
@@ -0,0 +1,71 @@
+package clear_all_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessEvent;
+import data_access.DataAccessTask;
+import data_access.WeekDataAccess;
+
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class ClearAllInteractorTest extends WeekDataAccess {
+    private final static ClearAllPresenter OUTPUT_BOUNDARY = new ClearAllPresenter();
+    private final static ClearAllDataAccess DS_GATEWAY = new ClearAllDataAccess();
+
+    @Test
+    void testClearInteractor() {
+        ClearAllInputBoundary inputBoundary = new ClearAllInteractor(OUTPUT_BOUNDARY, DS_GATEWAY);
+        ClearAllOutputData outputData = inputBoundary.clearAll();
+
+        assertTrue(outputData.getSuccess());
+        assertTrue(days.get(0).getTasks().isEmpty());
+        assertTrue(days.get(0).getEvents().isEmpty());
+
+        assertTrue(days.get(1).getTasks().isEmpty());
+        assertTrue(days.get(1).getEvents().isEmpty());
+
+        assertTrue(days.get(2).getTasks().isEmpty());
+        assertTrue(days.get(2).getEvents().isEmpty());
+
+    }
```

Comment:
Same issue here. All 6 days should be initialized with some Task and Event and then the clear() functionality should be tested.